pyiron_atomistics/lammps/base.py

```diff
@@ -593,28 +597,31 @@ def collect_output_log(self, file_name="log.lammps", cwd=None):
                      df['mean_pressure[5]'], df['mean_pressure[6]'], df['mean_pressure[3]']),
                     axis=-1,
                 ).reshape(-1, 3, 3).astype('float64')
-                pressures *= 0.0001  # bar -> GPa
                 if np.matrix.trace(rotation_matrix) != 3:
                     pressures = rotation_matrix.T @ pressures @ rotation_matrix
                 df = df.drop(
                     columns=df.columns[
                         (df.columns.str.startswith("mean_pressure") & df.columns.str.endswith(']'))
                     ]
                 )
-                df["mean_pressures"] = pressures.tolist()
-
-            generic_keys_lst = list(h5_dict.values()) + ["pressures", "mean_pressures", ]
+                pressure_dict["mean_pressures"] = pressures
+            generic_keys_lst = list(h5_dict.values())
             with self.project_hdf5.open("output/generic") as hdf_output:
                 # This is a hack for backward comparability
                 for k, v in df.items():
                     if k in generic_keys_lst:
-                        hdf_output[k] = np.array(v)
+                        # Convert to pyiron units
```

Comment:
Ok, final nit 😉 again this comment is now redundant, and if pycharm doesn't complain about the line length I'd try in-lining the array conversion as you do elsewhere.